assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/datesListTableHeader.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+import { DisplayDates } from '../../../../interfaces/datetimes/types';
+
+/**
+ * DatesListTableHeader
+ * header details for the Dates list table
+ */
+const datesListTableHeader = (displayDates: DisplayDates) => {
+	const cellsData = [
+		{
+			key: 'stripe',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-entity-list-status-stripe ee-rspnsv-table-column-micro',
+			value: '',
+		},
+		{
+			key: 'id',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-date-list-col-id ee-number-column ee-rspnsv-table-column-tiny',
+			value: __('ID', 'event_espresso'),
+		},
+		{
+			key: 'name',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-date-list-col-name ee-rspnsv-table-column-huge',
+			value: __('Name', 'event_espresso'),
+		},
+		{
+			key: 'start',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-date-list-col-name-start ee-rspnsv-table-column-default',
+			value: (
+				<>
+					<span className={'ee-rspnsv-table-long-label'}>{__('Start Date', 'event_espresso')}</span>
+					<span className={'ee-rspnsv-table-short-label'}>{__('Start', 'event_espresso')}</span>
+				</>
+			),
+		},
+		{
+			key: 'end',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-date-list-col-end ee-rspnsv-table-column-default',
+			value: (
+				<>
+					<span className={'ee-rspnsv-table-long-label'}>{__('End Date', 'event_espresso')}</span>
+					<span className={'ee-rspnsv-table-short-label'}>{__('End', 'event_espresso')}</span>
+				</>
+			),
+		},
+		{
+			key: 'capacity',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-date-list-col-capacity ee-rspnsv-table-column-tiny ee-number-column',
+			value: (
+				<>
+					<span className={'ee-rspnsv-table-long-label'}>{__('Capacity', 'event_espresso')}</span>
+					<span className={'ee-rspnsv-table-short-label'}>{__('Cap', 'event_espresso')}</span>
+				</>
+			),
+		},
+		{
+			key: 'sold',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-date-list-col-sold ee-rspnsv-table-column-tiny ee-number-column',
+			value: __('Sold', 'event_espresso'),
+		},
+		{
+			key: 'reserved',
+			type: 'cell',
+			className: 'ee-date-list-col-hdr ee-date-list-col-reserved ee-rspnsv-table-column-tiny ee-number-column',
+			value: (
+				<>
+					<span className={'ee-rspnsv-table-long-label'}>{__('Reserved', 'event_espresso')}</span>
+					<span className={'ee-rspnsv-table-short-label'}>{__('Rsrvd', 'event_espresso')}</span>
+				</>
+			),
+		},
```

Comment:
as mentioned in chat, this can be removed

Reply:
has been removed before the comment was written :)
https://github.com/eventespresso/event-espresso-core/pull/2500/commits/0a51bd2f3947ad5e14a779fc3ef6e32936b37690